tests/test_notify_suppliers_of_new_questions_answers.py

```diff
@@ -293,15 +284,28 @@ def test_send_emails_calls_mandrill_api_client(send_email, get_html_content):
         logger
     )
 
-    send_email.assert_called_once_with(
-        ['a@example.com', 'a2@example.com'],
-        "my content is bananas",
-        "MANDRILL_API_KEY",
-        EMAIL_SUBJECT,
-        EMAIL_FROM_ADDRESS,
-        EMAIL_FROM_NAME,
-        ["supplier-new-brief-questions-answers"],
-        logger=logger
+    assert send_email.call_count == 2
+    send_email.assert_any_call(
+        email_address="a@example.com",
+        template_name_or_id=mock.ANY,
+        template_personalisation={
+            "briefs": "https://www.digitalmarketplace.service.gov.uk/"
+                      "digital-outcomes-and-specialists/opportunities/3?utm_id=20170419qa"
+                      "\n"
+                      "https://www.digitalmarketplace.service.gov.uk/"
+                      "digital-outcomes-and-specialists/opportunities/4?utm_id=20170419qa",
+        }
+    )
+    send_email.assert_any_call(
+        email_address="a2@example.com",
+        template_name_or_id=mock.ANY,
+        template_personalisation={
+            "briefs": "https://www.digitalmarketplace.service.gov.uk/"
+                      "digital-outcomes-and-specialists/opportunities/3?utm_id=20170419qa"
+                      "\n"
+                      "https://www.digitalmarketplace.service.gov.uk/"
+                      "digital-outcomes-and-specialists/opportunities/4?utm_id=20170419qa",
+        }
```

Comment:
The test below this one uses a fake `MANDRILL_API_KEY` that should probably be updated to avoid confusion.

Reply:
Good spot